app/aws-lsp-json-binary/package.json

```diff
@@ -0,0 +1,29 @@
+{
+    "name": "@aws/lsp-json-binary",
+    "version": "0.0.1",
+    "description": "JSON Language Server Binary",
+    "main": "out/index.js",
+    "bin": {
+        "aws-lsp-json-binary": "./out/index.js"
+    },
+    "scripts": {
+        "compile": "tsc --build",
+        "package-x64": "pkg --targets node18-linux-x64,node18-win-x64,node18-macos-x64 --out-path bin --compress GZip .",
```

Comment:
not exactly related to this PR, but we should stop relying on `pkg`. Instead we should bundle it with webpack/rollup/etc. and run it with `node`